rskj-core/src/test/java/co/rsk/peg/PegUtilsGetTransactionTypeTest.java

```diff
@@ -35,7 +36,7 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-class BridgeSupportGetTransactionTypeTest {
+class PegUtilsGetTransactionTypeTest {
```

Comment:
Shouldn't this actually be testing PegUtilsLegacy? Since those are the old tests for getTransactionType method in BridgeSupport, that logic is now part of PegUtilsLegacy right?